biocrnpyler/polymer.py

```diff
@@ -187,8 +187,9 @@ def reverse(self):
 
 class NamedPolymer(OrderedPolymer):
```

Comment:
What do we use these for?

Reply:
When I create Polymer_transformations I "convert"  Constructs into NamedPolymers which is basically just a lossy version of copy.copy. A NamedPolymer has only the critical parts of a Construct such as the number of monomers and the name and whether its circular or not, but otherwise behave almost exactly the same as the Construct did.